### ISSUE TITLE
HOME : Events

### ISSUE THREAD

**Issue by skdwriting** [opensds/website#35]
I think the Events section before the testimonial and supporters look odd. It is not adding much to the intent of the home page.

Can we push down and give a nice film scroller below for the events with event name, location and a pic... (rather than so BIG pics?)

**Comment by stevenphtan** [comment]
We do want to promote events, right?

**Comment by anvithks** [comment]
> We do want to promote events, right?

Yes, we need to promote events but endorsement from key members / supporters will look better higher up in the page, IMHO.

**Comment by anvithks** [comment]
@stevenphtan @skdwriting Please let me know your thoughts on this.
